tests/staticfiles_tests/test_management.py

```diff
@@ -508,28 +508,30 @@ class TestCollectionOverwriteWarning(CollectionTestCase):
     # looking for was emitted.
     warning_string = "Found another file"
 
-    def _collectstatic_output(self, **kwargs):
+    def _collectstatic_output(self, verbosity=3, **kwargs):
         """
-        Run collectstatic, and capture and return the output. We want to run
-        the command at highest verbosity, which is why we can't
-        just call e.g. BaseCollectionTestCase.run_collectstatic()
+        Run collectstatic, and capture and return the output.
         """
         out = StringIO()
         call_command(
-            "collectstatic", interactive=False, verbosity=3, stdout=out, **kwargs
+            "collectstatic",
+            interactive=False,
+            verbosity=verbosity,
+            stdout=out,
+            **kwargs,
         )
         return out.getvalue()
 
     def test_no_warning(self):
         """
-        There isn't a warning if there isn't a duplicate destination.
+        There isn't a warning if there isn't a duplicate file.
         """
         output = self._collectstatic_output(clear=True)
         self.assertNotIn(self.warning_string, output)
 
-    def test_warning(self):
+    def test_warning_at_verbosity_2(self):
         """
-        There is a warning when there are duplicate destinations.
+        There is a warning when there are duplicate files at verbosity 2+.
```

Comment:
```suggestion
        There is a warning when there are duplicate destinations at verbosity
        2+.
```
Although we can also chop off this docstring, since the test name is self-documenting